src/org/openstreetmap/josm/gui/tagging/presets/TaggingPreset.java

```diff
@@ -39,16 +40,7 @@
 import org.openstreetmap.josm.command.Command;
 import org.openstreetmap.josm.command.SequenceCommand;
 import org.openstreetmap.josm.data.UndoRedoHandler;
-import org.openstreetmap.josm.data.osm.DataSet;
-import org.openstreetmap.josm.data.osm.IPrimitive;
-import org.openstreetmap.josm.data.osm.OsmData;
-import org.openstreetmap.josm.data.osm.OsmDataManager;
-import org.openstreetmap.josm.data.osm.OsmPrimitive;
-import org.openstreetmap.josm.data.osm.Relation;
-import org.openstreetmap.josm.data.osm.RelationMember;
-import org.openstreetmap.josm.data.osm.Tag;
-import org.openstreetmap.josm.data.osm.Tagged;
-import org.openstreetmap.josm.data.osm.Way;
+import org.openstreetmap.josm.data.osm.*;
```

Comment:
Quick comment: Don't use `*` imports. In IntelliJ Settings, go to `Editor` -> `Code Style` -> `Java` -> `Imports` and change `Class count to use import with '*'` and `Names count to use static import with '*'` to `999`.

While you are there, if you end up optimizing imports sometime, you should also change the `Import Layout` to be 
```
import static all other imports
<blank line>
import java.*
<blank line>
import javax.*
<blank line>
import all other imports
```

That really just minimizes the diff. It isn't wholly necessary, but it does avoid unnecessary code churn.

Running `ant pmd checkstyle` will find most of the code style issues.

```suggestion
import org.openstreetmap.josm.data.osm.DataSet;
import org.openstreetmap.josm.data.osm.IPrimitive;
import org.openstreetmap.josm.data.osm.OsmData;
import org.openstreetmap.josm.data.osm.OsmDataManager;
import org.openstreetmap.josm.data.osm.OsmPrimitive;
import org.openstreetmap.josm.data.osm.Relation;
import org.openstreetmap.josm.data.osm.RelationMember;
import org.openstreetmap.josm.data.osm.Tag;
import org.openstreetmap.josm.data.osm.Tagged;
import org.openstreetmap.josm.data.osm.Way;
```

Reply:
Thank you for the tip. 

The _checkstyle_ command is really helpful :)

Reply:
No problem. It is something that I frequently have to run (and sometimes I run, change something, and forget to run it again...)